internal/proto/query_coord.proto

```diff
@@ -213,6 +213,7 @@ message LoadMetaInfo {
   LoadType load_type = 1;
   int64 collectionID = 2;
   repeated int64 partitionIDs = 3;
+  // Deprecated
```

Comment:
`string metric_type=4  [deprecated=true];` 

The code generated by this grammar is more native.